primitives/src/types.rs

```diff
@@ -0,0 +1,191 @@
+/*
+   Copyright 2019 Supercomputing Systems AG
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+	   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*/
+
+//! Re-defintion of substrate primitives.
+//! Needed because substrate pallets compile to wasm in no_std.
+
```

Comment:
Maybe we can note that all these primitives are from the pallet-balances, aren't they?

Reply:
They are not. Should I paste a link from where they are taken?